### PR TITLE
Generate client using GraphQL schema

### PR DESCRIPTION
**Pull request by SaaldjorMike**

### CODE REVIEW

File: internal/api/status.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+func (s StatusResponse) IsDown() bool {
+	return s.Status != "OK" && s.Status != "WARN"
+}
+
+func (s StatusResponse) AtLeast(ver string) (bool, error) {
+	assumeLatest := true
+	version := strings.Split(s.Version, "-")
+	constraint, err := semver.NewConstraint(fmt.Sprintf(">= %s", ver))
+	if err != nil || len(version) == 0 {
+		return assumeLatest, fmt.Errorf("could not parse constraint of `%s`: %w", fmt.Sprintf(">= %s", ver), err)
```
[the remark this answers is not in the document]
Comment (SaaldjorMike) [reply internal/api/status.go:27]: Yes, `assumeLatest` is true at all times, but we don't always return it. At the end of the function we return:
```go
	return constraint.Check(semverVersion), nil
```
This is btw. a straight up copy/paste of https://github.com/humio/cli/blob/master/api/status.go into the new `internal/api` package, where the old package now includes deprecation warnings.

So overall, this is not any different from master.

File: internal/api/alerts.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+		TimeOfLastTrigger:  respUpdate.GetTimeOfLastTrigger(),
+		IsStarred:          respUpdate.GetIsStarred(),
+		Description:        respUpdate.GetDescription(),
+		ThrottleTimeMillis: respUpdate.GetThrottleTimeMillis(),
+		Enabled:            respUpdate.GetEnabled(),
+		Actions:            respUpdate.GetActions(),
+		Labels:             respUpdate.GetLabels(),
+		LastError:          respUpdate.LastError,
+		RunAsUserID:        runAsUserID,
+		QueryOwnershipType: *respQueryOwnership.GetTypename(),
```
[the remark this answers is not in the document]
Comment (SaaldjorMike) [reply internal/api/alerts.go]: I've pushed a commit to address this.

File: internal/api/status.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (bsocaciu) [reply internal/api/status.go:27]: Yep, i know it's a copy. I just wanted to make sure we're fixing possible issues if we have the chance

**Review by bsocaciu — APPROVED** [review]
Looks good to me now! 🚀

**Pull request merged**